includes/managers/class-fs-plan-manager.php

```diff
@@ -106,32 +106,59 @@ function has_free_plan( $plans ) {
 			return false;
 		}
 
-		/**
-		 * Find all plans that have trial.
-		 *
-		 * @author Vova Feldman (@svovaf)
-		 * @since  1.0.9
-		 *
-		 * @param FS_Plugin_Plan[] $plans
-		 *
-		 * @return FS_Plugin_Plan[]
-		 */
-		function get_trial_plans( $plans ) {
-			$trial_plans = array();
-
-			if ( is_array( $plans ) && 0 < count( $plans ) ) {
-				/**
-				 * @var FS_Plugin_Plan[] $plans
-				 */
-				for ( $i = 0, $len = count( $plans ); $i < $len; $i ++ ) {
-					if ( $plans[ $i ]->has_trial() ) {
-						$trial_plans[] = $plans[ $i ];
-					}
-				}
-			}
-
-			return $trial_plans;
-		}
+        /**
```

Comment:
@DanieleAlessandra please use tab for indentation since the rest of the file is also using tab.